shapes/typeIndex.shex

```diff
@@ -0,0 +1,13 @@
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+PREFIX typeIndex: <#>
+
+typeIndex:TypeIndex EXTRA a {
+    a [solid:TypeIndex] ;
+    a [solid:ListedDocument]
```

Comment:
I reckon `solid:UnlistedDocument` should be here as well?

Reply:
```suggestion
    ( a [solid:ListedDocument] | a [solid:UnlistedDocument] )
```